README.rst

```diff
@@ -224,6 +224,26 @@ debian
 
     An ephemeral container with which to run sysadmin tasks on the cluster
 
+get_db_name
+~~~~~~~~~~~
+
+    Get the database name (including the username, password, and port)
+
+get_media_name
+~~~~~~~~~~~~~~
+    
+    Get the pod's S3 media bucket name
+
+get_db_dump
+~~~~~~~~~~~
+
+    Get a dump of an environment's database
+
+load_db_dump
+~~~~~~~~~~~~
+
+    Load a database dump file into an environment's database
```

Comment:
If I needed to do this, I would probably be looking for a command with "restore" in the name rather than "load".

Reply:
went with `restore_db_from_dump`